insights/specs/default.py

```diff
@@ -544,8 +575,8 @@ def semid(broker):
     ls_run_systemd_generator = simple_command("/bin/ls -lan /run/systemd/generator")
     ls_R_var_lib_nova_instances = simple_command("/bin/ls -laR /var/lib/nova/instances")
     ls_sys_firmware = simple_command("/bin/ls -lanR /sys/firmware")
-    ls_usr_lib64 = simple_command("/bin/ls -lan /usr/lib64")
-    ls_usr_sbin = simple_command("/bin/ls -ln /usr/sbin")
+    # ls_usr_lib64 = simple_command("/bin/ls -lan /usr/lib64")
```

Comment:
@bfahr - Would you please hold off removing this spec, it might be used in a new rule.